src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      state.books.filter((book) => book.id !== bookId);
+    },
+  },
+});
+// console.log(booksSlice);
```

Comment:
- [ Required ] Good job implementing this project milestone, to make the project more professional please remove any pieces of commented code from your codebase. Commented code makes your codebase look unprofessional.

- [ Required ] I think it would be better if you also export the actions in `booksSlice` so that you can access them in your react components. Please consider exporting the actions as shown below.

```JavaScript
  export const { addBook, removeBook } = booksSlice.actions
```

Reply:
Thanks for pointing out and writing the code for me. I have added the code you suggested.  Also removed the comments.